handlers/command_handler.py

```diff
@@ -41,6 +41,7 @@ def _delete_first_two_words(self, phrase: str) -> str:
 
     async def command_ip(self, message):
         ip_help = """
+{0.author.mention}
```

Comment:
Due to a specific discord/discord.py bug, please use
```
<@{0.author.id}>
```
instead.

Reply:
Noted.